crates/wasmparser/src/readers/core/types.rs

```diff
@@ -776,6 +797,24 @@ pub struct StructType {
     pub fields: Box<[FieldType]>,
 }
 
+/// Represents a type of a continuation in a WebAssembly module.
+#[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
+pub struct ContType(pub PackedIndex);
+
+impl ContType {
+    /// Maps any `UnpackedIndex` via the specified closure.
+    #[cfg(feature = "validate")]
+    pub(crate) fn remap_indices(
+        &mut self,
+        map: &mut dyn FnMut(&mut PackedIndex) -> Result<()>,
+    ) -> Result<()> {
+        let mut idx = self.0;
+        map(&mut idx)?;
+        *self = ContType(idx);
+        Ok(())
+    }
+}
```

Comment:
If you'd like it's probably ok to omit this entirely and replace the caller with `map(&mut ty.0)?;`